infra/terraform/environments/dev/main.tf

```diff
@@ -234,4 +234,13 @@ module "service" {
       listener_rule_host_header = "ssweb.*"
     }
   }
+
+  job_definitions = {
+    "processQueue" = {
+      job_name = "processQueue",
+      command  = ["/var/www/html/vendor/bin/laminas --container=/var/www/html/config/container-cli.php", "queue:process-queue"],
```

Comment:
The command will always start with: `/var/www/html/vendor/bin/laminas --container=/var/www/html/config/container-cli.php` - could this be abstracted?